Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -1,46 +1,47 @@
 
-**Note:** If you just want to create and share explorations, you may be able to use the hosted server at https://www.oppia.org (in which case you don't need to install anything).
+> [!TIP]
+> If you just want to create and share explorations, you may be able to use the hosted server at https://www.oppia.org (in which case you don't need to install anything).
 
-*These installation instructions were last tested on 8 January 2024. For more information on issues that may occasionally arise with the installation process, see the [Troubleshooting](https://github.com/oppia/oppia/wiki/Troubleshooting) page or ask in the [GitHub Discussions](https://github.com/oppia/oppia/discussions).*
+> [!NOTE]
+> These installation instructions were last tested on 8 January 2024. For more information on issues that may occasionally arise with the installation process, see the [Troubleshooting](https://github.com/oppia/oppia/wiki/Troubleshooting) page or ask in the [GitHub Discussions](https://github.com/oppia/oppia/discussions).
 
-## Install prerequisites
+## 1. Install prerequisites
 
 Oppia relies on a number of programs and third-party libraries. Many of these libraries are downloaded automatically for you when you first run the `start.py` script provided with Oppia (see step 1 in the next section). However, there are some things that you will need to do beforehand:
 
 1. Make sure you have curl (used to download third-party libraries), git (which allows you to store the source in version control), unzip (used for unpacking zip files) and openjdk-11-jre (needed for GCP to work):
 
+   ```bash
+   sudo apt-get update
+   sudo apt-get install curl git unzip openjdk-11-jre
+   ```
+   
+   Alternatively, if you are on Debian/Ubuntu, you can use the `install_prerequisites.sh` script to install these. From the oppia directory, run:
 
-```
-sudo apt-get update
-sudo apt-get install curl git unzip openjdk-11-jre
-```
-
-Alternatively, if you are on Debian/Ubuntu, you can use the `install_prerequisites.sh` script to install these. From the oppia directory, run:
-
-```
-bash scripts/install_prerequisites.sh
-```
+   ```bash
+   bash scripts/install_prerequisites.sh
+   ```
 
 2. Verify that Java version 11+ is installed. You can use the command `java -version` to do this. (If you need to change the default binary for the `java` command on your machine, use `sudo update-alternatives --config java`.)
 
 3. Install Chrome from [Google's website](https://www.google.com/chrome). You'll need this to run tests.
 
 
-## Clone Oppia
+## 2. Clone Oppia
 
 1. Create a new, empty folder that will hold your Oppia work. Here, we call the folder `opensource`.
 
-2. Navigate to the folder (`cd opensource/`). Next, we'll [fork and clone](https://help.github.com/articles/fork-a-repo/) the Oppia repository.
+2. Navigate to the folder (`cd opensource/`). Next, we'll [fork and clone](https://help.github.com/articles/fork-a-repo/) the Oppia repository, as described in below steps.
```

Comment:
...in the steps below.

(Or just drop the 2nd sentence here and, in step 3, you can say "click on the `fork` button to fork Oppia's repository", where "fork Oppia's repository" is linked to https://help.github.com/articles/fork-a-repo/ .)